bazel-locks/Cargo.Bazel-default.lock

```diff
@@ -0,0 +1,41 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Is this directory (`bazel-locks/`) supposed to be checked in, or should it be in `.gitignore`?

Reply:
The files included are analogous to Cargo.lock

Made a directory because there are so many of them, I didn't want to clutter the top level directory.